asciidoc/modules/con_mg_migrating-applications-to-vertx-http-client.adoc

```diff
@@ -203,7 +203,8 @@ response.end(ar -> {
 ----
 +
 
-On the other hand, we can handle the response with some operations if we want to. You can see the difference in the next examples.
+On the other hand, you can also handle the response with operations like onSuccess, compose, bodyHandler, etc.
+The following examples demonstrate handling responses with the operation onSuccess :
 
```

Comment:
You can also handle the response with methods such as, `onSucces()`, `compose()`, and `bodyHandler()`.
The following examples demonstrate handling responses using the `onSuccess()` method.

Reply:
I would like to point here, that is not only those 3 methods, there are a few methods more.